### DISCUSSION TITLE
Proxy Consideration with Rancher

### DISCUSSION THREAD

**Discussion by KarimaFathi** [rancher/rke2#4601] [Q&A]
I would like to ask about a situation. I have successfully installed Rancher on an RKE2 cluster and everything appears to be functioning properly. However, I I didn't notice that the documentation specifies additional options to include during installation if operating behind a proxy server. Should I reinstall Rancher with the proxy options, or is it unlikely that I will encounter issues without them?

**Comment by brandond** [comment]
This isn't really an rke2 question, but if you need to use a proxy, I would probably recommend configuring your applications to do so?
